docs/reference/upgrade/rolling_upgrade.asciidoc

```diff
@@ -13,12 +13,16 @@ into the following two groups and upgrade the groups in this order:
 . Nodes that are not <<master-node,master-eligible>>. You can retrieve a list
 of these nodes with `GET /_nodes/_all,master:false` or by finding all the nodes
 configured with `node.master: false`.
+.. If you are using data tiers, you should
+upgrade the nodes by tier starting with frozen, cold, warm then hot-content tier.
```

Comment:
I think this could be misunderstood to allow to take any of frozen, cold and warm first as long as hot nodes are done last. Perhaps something like:

```suggestion
upgrade the nodes by tier, completing one tier at a time, in this order: frozen, cold, warm and finally hot tier.
```